task/control-plane-agent/src/mgs_gimlet/host_phase2.rs

```diff
@@ -181,11 +180,23 @@ impl HostPhase2Requester {
 
         let n = gateway_messages::serialize(tx_buf, &message).unwrap_lite();
 
+        let vid = {
+            let target = match port {
+                SpPort::One => task_net_api::SpPort::One,
+                SpPort::Two => task_net_api::SpPort::Two,
+            };
+            task_net_api::VLANS
+                .iter()
+                .find(|v| v.port == target)
+                .unwrap()
```

Comment:
Please use `unwrap_lite` if this must panic. (An `EnumMap` might be nice here if you expect there to always be a mapping for every variant in `SpPort`...)

Reply:
Also fixed in the Grand VLAN Refactoring